server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/test/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRoutesTest.java

```diff
@@ -1849,6 +1918,189 @@ void exportShouldSaveDeletedMessagesDataToBlobStore() throws Exception {
         }
     }
 
+    @Nested
+    class PurgeTest {
+
+        @Test
+        void purgeShouldReturnATaskCreated() {
+            given()
+                .queryParam("action", "restore")
+                .body(MATCH_ALL_QUERY)
+            .when()
+                .post(BOB_PATH)
+            .then()
+                .statusCode(HttpStatus.CREATED_201)
+                .body("taskId", notNullValue());
+        }
+
+        @Test
+        void purgeShouldProduceASuccessfulTaskWithAdditionalInformation() {
+            vault.append(USER, DELETED_MESSAGE, new ByteArrayInputStream(CONTENT)).block();
+            vault.append(USER, DELETED_MESSAGE_2, new ByteArrayInputStream(CONTENT)).block();
+
+            String taskId =
+                given()
```

Comment:
`given() ... when() ... then()` is for when you need to assert the answer. If not, better use `with()` syntax for requests we don't need to assert the result